src/fit-bounds.js

```diff
@@ -37,12 +69,12 @@ export default function fitBounds({
     Math.abs(se[1] - nw[1])
   ];
   const center = [
-    (se[0] + nw[0]) / 2,
-    (se[1] + nw[1]) / 2
+    (se[0] + nw[0]) / 2 + lateralOffset,
+    (se[1] + nw[1]) / 2 + verticalOffset
```

Comment:
`lateralOffset` and `verticalOffset` are in pixels at the new zoom.
`se` and `nw` are in pixels at the old zoom.